syntax = "proto3";

package cafm.identity.ou;

message OuAppService_ouByNameRequest {
  string name = 1;
}

message OuAppService_ouByNameResponse {
  string id = 1;
  string name = 2;
}

service OuAppService {
  rpc ouByName (OuAppService_ouByNameRequest) returns (OuAppService_ouByNameResponse) {}
}

